pygmt/base_plotting.py

```diff
@@ -303,20 +303,55 @@ def grdcontour(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
-        W="pen",
+        A="img_out",
         B="frame",
-        I="shading",
         C="cmap",
+        D="img_in",
+        E="dpi",
+        G="bit_color",
+        I="shading",
+        J="projection",
+        M="monochrome",
+        N="noclip",
```

Comment:
`noclip` or `no_clip`? GMT.jl supports both. Perhaps the latter is more Pythonic and readable? Note that many other modules (e.g. `plot`, `text`) also have the **-N** (`noclip` or `no_clip`) option.

Reply:
Ok, snake case it is then (i.e. `no_clip`). Let's follow https://www.python.org/dev/peps/pep-0008/#naming-conventions.